web/src/components/Required.tsx

```diff
@@ -1,22 +1,8 @@
 import * as React from 'react'
+import { Tag } from './Tag'
 
 const color = '#f59e0b'
 
 export function Required() {
```

Comment:
Yeah you could probably organize these components under `Tag` and also rename them to InternalTag, RequiredTag, ... .